### PR TITLE
Update Reference Implementation to support Spark 3

### PR DESCRIPTION
**Pull request by hallihan**
- Update project files to build for Spark 3
- Update README to us latest LTS version of Azure Databricks
- Remove old embedded version of the spark monitoring library and link to current repo

### CODE REVIEW

**Comment by hallihan** [comment]
We'll hold off on merging this until https://github.com/MicrosoftDocs/architecture-center-pr/pull/3812 is live

File: README.md [hunk 1]
```diff
@@ -338,14 +302,10 @@ For this section, you require the Log Analytics workspace ID and primary key. Th
...
 
-1. Under **Dependent Libraries** click **Add** again.
-
-1. Change **Library Source** to **DBFS/ADLS**, confirm that Library Type is **Jar** and enter `dbfs:/azure-databricks-job/azure-databricks-monitoring-0.9.jar` in the **File Path** text box and select **Add**.
-
 1. In the **Parameters** field, enter the following (replace **\<Cosmos DB Cassandra host name\>** with a value from above):
 
     ```shell
-    ["-n","jar:file:/dbfs/azure-databricks-job/cb_2020_36_cousub_500k.zip!/cb_2020_36_cousub_500k.shp","--taxi-ride-consumer-group","taxi-ride-eh-cg","--taxi-fare-consumer-group","taxi-fare-eh-cg","--window-interval","1 minute","--cassandra-host","<Cosmos DB Cassandra host name>"]
+    ["-n","jar:file:/dbfs/azure-databricks-job/cb_2020_36_cousub_500k.zip!/cb_2020_36_cousub_500k.shp","--taxi-ride-consumer-group","taxi-ride-eh-cg","--taxi-fare-consumer-group","taxi-fare-eh-cg","--window-interval","1 hour","--cassandra-host","<Cosmos DB Cassandra host name>"]
```
Comment (ckittel) [inline README.md:308]: Unrelated to this change, but the GitHub UI won't let me note it.... a few lines down.

> 1. Select the **Runs** tab and lick **Run Now**.

Always _lick_ **Run Now** buttons when you can.  #ProTip

**Review by ckittel — COMMENTED** [review]
I literally know nothing about this area (me -> 🤡), but I did a quick glance through and my only contribution to this conversation is about the hygiene of licking buttons on the Azure Portal. How pathetic of me.

The real experts will show up to provide a real review.   :)

File: README.md (hunk 1 above)
Comment (hallihan) [reply README.md:308]: Ha!  I'll fix that.  Don't need to be trying out new UI interactions like that...

Comment (ckittel) [reply README.md:308]: I mean, global hackathon month is here ... let's try.  Might be the next big thing. Flavorful User Interfaces

**Review by skabou — APPROVED** [review]

**Pull request merged**